pkgs/tools/typesetting/soupault/default.nix

```diff
@@ -9,35 +9,36 @@
 let
   pname = "soupault";
 
-  version = "4.6.0";
+  version = "4.7.0";
 in
 ocamlPackages.buildDunePackage {
   inherit pname version;
 
   minimalOCamlVersion = "4.13";
 
-  duneVersion = "3";
+  flambdaSupport = true;
```

Comment:
This is dead code, right?

Reply:
One of y’all recommended me no put the dune version in a different repository because the default is 3 already?

Flambda is to my understanding an optimization, & it compiles/works so it seems like the right thing to include, or am I misunderstanding something?

Reply:
Removing the `duneVersion` attribute is fine.

My remark was about the `flambdaSupport` attribute which is neither used by `buildDunePackage` nor by `mkDerivation`.

Reply:
oooh